doozer/doozerlib/constants.py

```diff
@@ -8,6 +8,7 @@
 
 GITHUB_TOKEN = "GITHUB_TOKEN"
 BREWWEB_URL = "https://brewweb.engineering.redhat.com/brew"
+BREWHUB_URL = "https://brewhub.engineering.redhat.com/brewhub"
```

Comment:
I don't think we want to hardcode this, instead use https://github.com/openshift-eng/ocp-build-data/blob/e7585b181712a17f548e6601ce434f1e0d08299e/group.yml#L90

Reply:
fixed in https://github.com/openshift-eng/art-tools/pull/51/commits/906d84751f15b00b02083b9cfe428b05c0888896